maven-plugin/src/main/resources/settings.xml

```diff
@@ -24,10 +24,22 @@
             </properties>
             <repositories>
                 <repository>
-                    <id>openmrs-repo</id>
+                    <id>openmrs repo</id>
```

Comment:
This should still be `openmrs-repo`

Reply:
Sorry @ibacher   for late reply my computer had got issues but am now settled thanks for this